doc/examples/segmentation/plot_rolling_ball.py

```diff
@@ -0,0 +1,151 @@
+"""
+======================================================
+Background Subtraction using the Rolling Ball Method
+======================================================
+
+The rolling ball filter is a segmentation method that aims to separate the
+background from a grayscale image in case of uneven exposure. It is frequently
+used in biomedical image processing and was first proposed by Stanley R.
+Sternberg (1983) in the paper Biomedical Image Processing [1]_.
+
+The idea of the algorithm is quite intuitive. We think of the image as a
+surface that has unit-sized blocks stacked on top of each other for each
+pixel. The number of blocks is determined by the intensity of a pixel. To get
+the intensity of the background at a desired position, we imagine submerging a
+ball into the blocks at the desired pixel position. Once it is completely
+covered by the blocks, the height of the ball determines the intensity of the
+background at that position. We can then *roll* this ball around below the
+surface to get the background values for the entire image. As Sternberg
+puts it: "We can visualize a solid sphere that moves freely within the solid
+volume of the gel image umbra but is constrained by the upper surface of the
+umbra."
+
+.. [1] Sternberg, Stanley R. "Biomedical image processing." Computer 1 (1983):
+    22-34. :DOI:`10.1109/MC.1983.1654163`
+
+To implement this algorithm in skimage we can use the following:
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import morphology
+from skimage import data
+from skimage.util import invert
+
+
+def rolling_ball(image, radius=30, white_background=False):
```

Comment:
how about allowing `radius` to be a tuple. the tuple could then be interpreted as (xy_radius, intensity_radius) or (x_radius, y_radius, intensity-_radius). I would probably convert it to a tuple of length three in the beginning and then use the tuple throughout.